system-test/integration_test.sh

```diff
@@ -23,13 +23,9 @@ export GCLOUD_TESTS_NODEJS_PROJECT_ID="cloud-profiler-e2e"
 export GCLOUD_TESTS_NODEJS_ZONE="us-east4-b"
 export GOOGLE_APPLICATION_CREDENTIALS="${SERVICE_KEY}"
 
-# Move test to go path.
-export GOPATH="$HOME/go"
-mkdir -p "$GOPATH/src"
-cp -R "system-test" "$GOPATH/src/proftest"
-
 # Run test.
-cd "$GOPATH/src/proftest"
+cd "system-test"
+go mod init
```

Comment:
Maybe add a comment to clarify what this achieves.

Reply:
Done.